subt/tools/allsync.py

```diff
@@ -7,14 +7,24 @@
 
 
 def main():
-    robots = {
+    import argparse
+    parser = argparse.ArgumentParser(description='rsync logs from robots')
+    parser.add_argument('robots', nargs='*', choices=['eduro', 'k2', 'k3', 'mobos', 'maria', []], default=[])
```

Comment:
optionally you could use all_robots.keys() as choices ... BTW is [] needed?

Reply:
done